CodePushConfig.m

```diff
@@ -31,6 +33,14 @@ - (instancetype)init
     NSString *deploymentKey = [infoDictionary objectForKey:@"CodePushDeploymentKey"];
     NSString *serverURL = [infoDictionary objectForKey:@"CodePushServerURL"];
     
+    NSUserDefaults* userDefaults = [NSUserDefaults standardUserDefaults];
+    NSString* clientUniqueId = [userDefaults stringForKey:ClientUniqueIDConfigKey];
```

Comment:
It is an iOS bug. identifierForVendor sometimes changes when an app is updated. http://stackoverflow.com/a/33130588/2670678

I have seen other plugins such as cordova-plugin-device that also persist the value to workaround this problem. Seems like its safer to just keep this.